### DISCUSSION TITLE
Account suspended

### DISCUSSION THREAD

**Discussion by sefinek-actions** [community/community#144564] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

My account was suspended because I used a VPN to bypass my internet provider's routing, which limits me to a maximum of 200 KB/s when cloning repositories. With a VPN, I get over 20 MB/s.

As a result, my account has been suspended, and visiting my profile @sefinek leads to a 404 error. I have submitted a ticket - will I even get a response? @github-staff

Thanks.

**Comment by mecodeatlas** [comment]
Hi @sefinek-actions 
Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
